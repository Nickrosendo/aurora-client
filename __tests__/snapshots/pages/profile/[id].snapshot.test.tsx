import React from 'react';
import renderer from 'react-test-renderer';
import ProfileDetail from '@root/pages/profile/[id]';

Object.defineProperty(window, 'matchMedia', {
  writable: true,
  value: jest.fn().mockImplementation((query) => ({
    matches: false,
    media: query,
    onchange: null,
    addListener: jest.fn(), // Deprecated
    removeListener: jest.fn(), // Deprecated
    addEventListener: jest.fn(),
    removeEventListener: jest.fn(),
    dispatchEvent: jest.fn(),
  })),
});

jest.mock('@root/components', () => ({
  ThemeContainer: jest.fn().mockImplementation(() => <>ThemeContainer</>),
  Header: jest.fn().mockImplementation(() => <>Header</>),
}));

it('should render profile detail matching snapshot', () => {
  const tree = renderer.create(<ProfileDetail />).toJSON();
  expect(tree).toMatchSnapshot();
});
