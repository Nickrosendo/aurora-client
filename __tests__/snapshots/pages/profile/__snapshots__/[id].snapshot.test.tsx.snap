// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render profile detail matching snapshot 1`] = `"ThemeContainer"`;
